recipes/pretextsnapshot/meta.yaml

```
@@ -0,0 +1,33 @@
+{% set name = "PretextSnapshot" %}
+{% set version = "0.0.2" %}
+{% set checksum = "bbb4ce292c13263616b24233f49859cb9d139cd84d56729ec5dd495782d87536" %}
+{% set about = "Commandline image generator for Pretext contact maps." %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} # [linux64]
+    - make
+    - clang>=9.0.0
```

This should at least be made osx only (so  add `# [osx]`). Do you really need this new a version of clang?

---

I use certain intrinsics that are only available in later versions of clang

---

Can you turn that into
```suggestion
    - clang_osx-64 >=9.0.0
```
? Otherwise we won't get the usual compiler setup ("compiler activation scripts").

---

Another possibility could be to add a `conda_build_config.yaml` alongside `meta.yaml` with this content
```yaml
cxx_compiler:          # [osx]
  - clangxx            # [osx]
cxx_compiler_version:  # [osx]
  - 9                  # [osx]
```
However, I'm not entirely sure if that'll work.

---

I also use clang to build on linux

---

Better to use gcc on Linux for consistency with other packages.